com.raywenderlich.swiftlint.yml

```diff
@@ -43,24 +37,14 @@ opt_in_rules:
   - redundant_nil_coalescing
   - redundant_type_annotation
   - strict_fileprivate
+  - todo
   - toggle_bool
-  - trailing_closure
   - unneeded_parentheses_in_closure_argument
   - unused_import
   - vertical_whitespace_closing_braces
   - vertical_whitespace_opening_braces
   - yoda_condition
 
-
-custom_rules:
-  array_constructor:
-    name: "Array/Dictionary initializer"
-    regex: '[let,var] .+ = (\[.+\]\(\))'
-    capture_group: 1
-    message: "Use explicit type annotation when initializing empty arrays and dictionaries"
-    severity: warning
```

Comment:
Because:
<img width="780" alt="Screen Shot 2021-03-04 at 12 08 51" src="https://user-images.githubusercontent.com/5990580/109955128-6e4e2400-7ce2-11eb-8b9e-656e5d2300d5.png">

Reply:
🤦‍♀️